r/tools/winlibs.R

```diff
@@ -37,28 +37,32 @@ if (!file.exists(sprintf("windows/arrow-%s/include/arrow/api.h", VERSION))) {
         silent = quietly
       )
     }
+
     # URL templates
     nightly <- paste0(
       getOption("arrow.dev_repo", "https://nightlies.apache.org/arrow/r"),
       "/libarrow/bin/windows/arrow-%s.zip"
     )
-    rwinlib <- "https://github.com/rwinlib/arrow/archive/v%s.zip"
```

Comment:
@nealrichardson let me know if you want to keep rwinlib and add the artifactory as an  additional step

Reply:
Yeah I think we want to keep rwinlib as the first check, it makes sense for us to stay plugged into that project even if it's not completely necessary anymore.